packages/vue-components/src/Submenu.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <li
+    ref="submenu"
+    :class="[addClass, 'dropdown-submenu',
+             { 'dropright': dropright, 'dropleft': dropleft }]"
+  >
+    <slot name="button">
+      <a
+        class="submenu-toggle"
+        role="button"
+        :class="{disabled: disabled}"
+      >
+        <slot name="_header">
+          <slot name="header"></slot>
+        </slot>
+      </a>
+    </slot>
+    <slot name="dropdown-menu">
+      <ul class="dropdown-menu">
+        <slot></slot>
+      </ul>
+    </slot>
+  </li>
+</template>
+
+<script>
+import { toBoolean } from './utils/utils';
+import $ from './utils/NodeList';
+import positionSubmenu from './utils/submenu';
+
+export default {
+  props: {
+    addClass: {
+      type: String,
+      default: '',
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      show: false,
+      dropright: true,
+      dropleft: false,
+    };
+  },
+  computed: {
+    disabledBool() {
+      return toBoolean(this.disabled);
+    },
+    showBool() {
```

Comment:
this (could replace with just `show`) and `slots()` dosen't seem used

Reply:
Thanks!. Replaced all `showBool` with `show`